tests/commands/package/test_call.py

```diff
@@ -758,3 +762,149 @@ def test_already_packaged(package_command, first_app, tmp_path):
     # package_app() call, no new artefact is created; the absence
     # of this file shows that the old one has been deleted.
     assert not artefact_path.exists()
+
+
+def test_package_app_single(package_command, first_app):
+    """If the --app flag is used, only the selected app is packaged."""
+    # Add a single app
+    package_command.apps = {
+        "first": first_app,
+    }
+
+    # Configure the --app option
+    options, _ = package_command.parse_options(["--app", "first"])
```

Comment:
As with the build test - this test should be parameterised to confirm both `--app` and `-a` work as arguments.